Tests/.swiftlint.yml

```diff
@@ -0,0 +1,2 @@
+disabled_rules:
+  - line_length
```

Comment:
This utilizes the change in SwiftLint 0.22:

> Nested configurations will now be merged with parent configurations rather than replace them outright.